admin_manual/configuration/big_file_upload_configuration.rst

```diff
@@ -26,8 +26,10 @@ System Configuration
 Configuring Your Webserver
 --------------------------
 
-ownCloud comes with its own ``owncloud/.htaccess`` file. Set the following 
-two parameters inside this ``.htaccess`` file::
+| ownCloud comes with its own ``owncloud/.htaccess`` file.
+| If PHP-FPM is used, it can't read ``.htaccess`` PHP settings unless a PECL extension is installed. If PHP-FPM is used without the PECL extension installed, settings and permissions must be set in the ``owncloud/.user.ini`` file.
```

Comment:
These linebreaks in the RST source are made because it shortens the lines and make diffs easier to read on github. Because long lines aren't wrapped here. It's just a convention for this documentation.